chucks-pick3-web/angular-app/src/app/components/customer-summary.component/customer-summary.component.ts

```diff
@@ -1,15 +1,77 @@
-import { Component, OnInit } from '@angular/core';
+import {Component, OnInit} from '@angular/core';
+import {CustomerInfo} from 'src/app/models/customer-info';
+import {CustomerInfoModel} from '../../models/customer-info.model';
+
 
 @Component({
   selector: 'app-customer-summary',
   templateUrl: './customer-summary.component.html',
   styleUrls: ['./customer-summary.component.css']
 })
 export class CustomerSummaryComponent implements OnInit {
+  public customers: CustomerInfo [] = [];
 
-  constructor() { }
+  constructor() {
+  }
 
   ngOnInit() {
+    this.customers = [
+      {
+        firstName: 'David',
+        lastName: 'Ocampo',
+        email: 'docampo@bytepushers.software',
+        phone: '555-555-5555',
+        city: 'Denton',
+        state: 'Texas'
+      },
+      {
+        firstName: 'Mark',
+        lastName: 'Caroll',
+        email: 'mcarroll@gmail.com',
+        phone: '555-555-5555',
+        city: 'Elko',
+        state: 'Nevada'
+      },
+      {
+        firstName: 'George',
+        lastName: 'Anderson',
+        email: 'ganderson@gmail.com',
+        phone: '555-555-5555',
+        city: 'Los Angeles',
+        state: 'California'
+      },
+      {
+        firstName: 'Henry',
+        lastName: 'Smith',
+        email: 'hsmith@gmail.com',
+        phone: '555-555-5555',
+        city: 'Boston',
+        state: 'Massachusetts'
+      }
+    ];
   }
 
+
+  public deleteRow(customer) {
+    if (customer !== null && customer !== undefined) {
+      this.customers.splice(customer, 1);
```

Comment:
Here you need to find the `customer` that was passed in via the `deleteRow(customer) method` in the `this.customers array`.  Use the `customer.id` property to determine if you found the correct customer.

Reply:
Completed id match for customer, deleteRow now deletes desired customer.